docs/data/data-grid/filtering-recipes/QuickFilterOutsideOfGrid.js

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import { Portal } from '@mui/base/Portal';
+import Portal from '@mui/material/Portal';
```

Comment:
I would expect to see this here

```suggestion
import { Portal } from '@base_ui/react/Unstable_Portal';
```

Reply:
But when do you expect to see this?
I think everybody agrees that, at some point, we will use `@base_ui/react` whenever possible, but I think nobody (in X at least) have any idea if we should change the imports today, or in one year.
The migration period is not really clear tbh.

Reply:
The root of the feedback is that it feels backward to invest energy to depend on @mui/material back, when we could invest the same energy to depend on @base_ui/react. As a Base UI user, I wouldn't understand why I'm asked to replace my imports to use Material UI and have no more docs for those API, when there is a v2 of Base UI in the making, why not ask me to move to this v2?

Reply:
We agreed to revert the changes going from @mui/base to @mui/material, those will be reverted (especially consdiering they are changing packages that don't have Material UI in their dependency).

Reply:
> The sooner the better 😄. The root of the feedback is that it feels backward to invest energy to depend on @mui/material back for things that are part of the Base UI product scope. Instead, we could invest this energy to depend on @base_ui/react. So why not?

We don't want @base_ui/react to be used by other stable packages until we're pretty sure we've got the API right (= we're at least in beta). Let's make Base UI ready to use first, and then we can progressively migrate all the other libraries to use it. Consuming Base UI in its current shape by other libraries will limit our ability to move forward quickly and risk the stability of these libraries.

Reply:
Michal's comment here makes sense. We have just been working on Base UI for ~5 months, we haven't even released a public alpha yet. We cannot have stable libs using Base UI at this point.

Furthermore, using @base_ui/react as a dumping ground for legacy code will dilute the credibility of the lib, and cause confusion for new Base UI users we're trying to attract.

Reply:
> As a user, I wouldn't understand why I'm asked to replace my imports to use Material UI and have no more docs for those API, and I don't plan to use those default MD style, when there is a v2 of Base UI in the making, why not ask me to move to this v2 even if it's under a legacy unstable_ flag? But like, either we tell people to use the Portal component, or to use the React Portal API directly, in both cases, this feels like something that we should handle in Base UI. Same for click away, same for auto size, etc. Overall, I'm pushing so we optimize for each component, as isolated product scopes.

I think this is out of scope for this effort. To conclude the discussion, we could depend on @mui/mateiral so that future fixes/improvements can be  included in MUI X as well, or keep depending on @mui/base and be aware that the Core team does not plan to keep doing changes in this package. This is the only decision that MUI X needs to make. cc @alexfauquette

Reply:
This is the last decision we need to make in order to merge this PR. We kicked off too many other discussions that are not relevant for the changes. I mean, we could have had this change without opening this PR, nothing that is publicly available changed on Core, I was just playing a good citizen by porting these changes :D

Reply:
Using `import Portal from '@mui/material/Portal';` is good for me. It's effectively a step backward, but a very small one.